src/worker.js

```diff
@@ -0,0 +1,22 @@
+const { readFileSync, writeFileSync } = require('fs');
+
+process.on('message', ({ filesChunk, classesMapping, mode }) => {
+
+  console.log(`${mode}-jetifying ${filesChunk.length} file(s)...`);
+
+  for (const file of filesChunk) {
+    let data = readFileSync(file, { encoding: 'utf8' });
+    for (const [oldClass, newClass] of classesMapping) {
+      if (data.includes(mode === 'forward' ? oldClass : newClass)) {
+        data = mode === 'forward' ?
+          data.replace(new RegExp(oldClass, 'g'), newClass) :
+          data.replace(new RegExp(newClass, 'g'), oldClass);
+        writeFileSync(file, data, { encoding: 'utf8' });
+      }
+    }
+  }
+
+  console.log(`${mode}-jetifyied ${filesChunk.length} file(s)`);
```

Comment:
should just be jetified I think? Because…english 😆 (rule, who needs rules, we like exceptions)

Reply:
Haha yeah my mistake! it should be `jetified`... We are adding to both React Native and the dictionary